sql/pg_tier.sql

```diff
@@ -148,23 +152,22 @@ BEGIN
   WHERE cred_id = 1;
 
   IF NOT FOUND THEN
-    RAISE object_not_in_prerequisite_state USING MESSAGE = 'Tier Credentials Not Found';
+    RAISE object_not_in_prerequisite_state
+     USING MESSAGE = 'No Object Store Credentials Found',
+     HINT = 'To set credentials call set_tier_config() function';
```

Comment:
I think include the schema.

```suggestion
     HINT = 'To set credentials call tier.set_tier_config() function';
```